docs/guides/getting-started-orchestrate-human-tasks.md

```diff
@@ -100,13 +139,14 @@ To run this guide, make sure to be in **Implement** mode to specify the technica
 
 1. A **start event** is automatically added to the canvas. Click it to display configuration and append options.
```

Comment:
The start event should be automatically selected- you can check this behavior on SaaS